ports/esp32/machine_touchpad.c

```diff
@@ -128,11 +193,13 @@ static mp_obj_t mtp_config(mp_obj_t self_in, mp_obj_t value_in) {
     esp_err_t err = touch_pad_config(self->touchpad_id, value);
     #elif SOC_TOUCH_SENSOR_VERSION == 2
     esp_err_t err = touch_pad_config(self->touchpad_id);
+    #elif SOC_TOUCH_SENSOR_VERSION == 3
+    esp_err_t err = ESP_OK;
```

Comment:
If this function doesn't do anything then we should `#ifdef` it out and not include it on the object for version 3